bittensor/commands/misc.py

```diff
@@ -52,93 +71,3 @@ def add_args(parser: argparse.ArgumentParser):
         )
 
         bittensor.subtensor.add_args(update_parser)
-
-
-class ListSubnetsCommand:
```

Comment:
Are we disabling the list subnet command? I thought it was quite useful to list all of the subnets at a quick glance

Reply:
No, this is a duplicate. :D